### PR TITLE
First part of polish translation

### PR DESCRIPTION
**Pull request by kisielwrosole**
Hi, 
It's first part of translation, I will be translating it further, but it's part with basic usage.

### CODE REVIEW

**Comment by jonataslaw** [comment]
Thank you very much for your contribution!
I appreciated that a lot.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![](get.png)
 
-*Languages: English (this file), [Brazilian Portuguese](README.pt-br.md).*
+*Languages: [English](README.md), [Brazilian Portuguese](README.pt-br.md), [Polish](README.pl.md).*
```
Comment (Nipodemos) [inline README.md:3]: In my opinion, it makes little sense to put a link to the same readme you are already in. If i am already reading english readme, why do i need a link to this same english readme?

File: README.pt-br.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ![](get.png)
 
-*Idiomas: [Inglês](README.md), Português Brasileiro (este aquivo)*
+*Idiomas: [English](README.md), [Brazilian Portuguese](README.pt-br.md),[Polish](README.pl.md).*
```
Comment (Nipodemos) [inline README.pt-br.md:3]: Same here, If i am already reading portuguese readme, why do i need a link to this same portuguese readme?

File: README.md (hunk 1 above)
Comment (jonataslaw) [reply README.md:3]: I agree with that

**Comment by Nipodemos** [comment]
hey, if you make those little changes, he can merge your PR! I think it would be nice to have another translator around 😄

**Comment by Nipodemos** [comment]
so sad that this guy disappeared... ☹️

**Comment by kisielwrosole** [comment]
Sorry, but my absence was made by the end of semester. I will correct it soon and get it up to date with rest readme files ;)

**Comment by Nipodemos** [comment]
@kisielwrosole i'm glad that you are back 😄

**Comment by Nipodemos** [comment]
hey, you forgot other files 😅

**Comment by malcolmpl** [comment]
Please, do not merge until the translation will be done correctly. It seems that the translation was mainly done using some translation engine. Reading that is a nightmare, names which should remain English are poorly translated and few hand-translated lines have grammatical mistakes.

**Pull request closed without merging**